game/missiongenerator/aircraft/waypoints/landrefuel.py

```diff
@@ -4,7 +4,7 @@
 from .pydcswaypointbuilder import PydcsWaypointBuilder
 
 
-class StopoverBuilder(PydcsWaypointBuilder):
+class LandRefuelBuilder(PydcsWaypointBuilder):
```

Comment:
Unused after other comments are addressed, I think

Reply:
it is not. see above